src/renderer/store/handlers/channel.ts

```diff
@@ -199,7 +211,7 @@ const sendTypingIndicator = value => async (dispatch, getState) => {
 
   if (useTor && users[channel.id] && users[channel.id].onionAddress) {
     try {
-      const memo = await packMemo(message, value)
+      const memo = await packMemo(message, value, onionAddressProp, zcashAddressProp)
```

Comment:
You have also added `onionAddress` and `zcashAddress` to `DisplayableMessage`. So you should pass them to 
`messages.createMessage` so that they are included in the message rather than passing them to `packMemo()`